Assets/MixedRealityToolkit.SDK/Features/UX/Interactable/Scripts/Interactable.cs

```diff
@@ -989,18 +1131,111 @@ protected int GetVoiceCommandIndex(string command)
 
         #endregion VoiceCommands
 
+        #region NearInteractionHandlers
+        /// <summary>
+        /// Is the eventData a grip action or near pointer?
+        /// </summary>
+        /// <param name="eventData"></param>
+        /// <returns></returns>
+        protected bool IsGrip(IMixedRealityPointer pointer, MixedRealityInputAction action)
+        {
+            if (action.Description.ToLower().IndexOf("grip") > -1 || pointer as IMixedRealityNearPointer != null)
```

Comment:
Careful, we are highly discouraging people to do direct string comparisons to determine if a pointer is a grip. Instead, the pattern we try to recommend is expose a mixedrealityaction gripAction, and set that to "Grip". @luis-valverde-ms  FYI as another example of this pattern.

Reply:
I was hoping that's the case, it's really problematic as is. One issue I thought about was users could create their own actions so they could name them anything, so going by name or existing actions could break. On the MR controller side, we would get a mixed reality action for grip, but for HoloLens, we just get Select, but at least we can cast to a near pointer to check intent. I'll reach out to @luis-valverde-ms

Reply:
I'm putting this as a TODO, will research later.